images/build/benv.sh

```diff
@@ -208,8 +208,8 @@ benv-resolve() {
       return 1
     fi
 
-    local SDK_DIR=$(readlink $"$runtimeDir/sdk")
-
+    local sdkVersion=$(cat "$runtimeDir/sdkVersion.txt" | tr -d '\r') 
```

Comment:
we try to find the dotnet cli (i.e dotnet executable) from the information in the metadata file